.dockerignore

```diff
@@ -0,0 +1,138 @@
+# Created by https://www.gitignore.io/api/flask
+# Edit at https://www.gitignore.io/?templates=flask
```

Comment:
Building a `.dockerignore` based on your `.gitignore` is often a really sensible starting point, but beware they do have slightly different syntaxes which can catch you out! I'll admit I always forget the details, but I believe the one that most catches me out is the [relative paths one discussed here](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/); particularly for sensitive files it's worth having a quick sanity check that this will still cover it